reference/HA.md

```diff
@@ -1,7 +1,12 @@
 ## High Availability 
 
-Mayastor 2.0 brings in the High Availability (HA) feature. In the event of a failure, this feature will guarantee I/O continuity. 
-The HA feature consists of two components: the HA node agent and the cluster agent. The HA node agent looks for io-path failures from applications to their corresponding targets. If any such broken path is encountered, the HA node agent informs the cluster agent. The cluster-agent then creates a new target on a different (live) node. Once the target is created, the `csi-node` establishes a new path between the application and its corresponding target. The HA feature restores the broken path within seconds, ensuring negligible downtime. 
+Mayastor 2.0 brings in the High Availability (HA) feature. In the event of a failure, this feature attempts to ensure I/O continuity. 
```

Comment:
Mayastor 2.0 enhances High Availability (HA) of the volume target with the nexus switch-over feature. In the event of the target failure, the switch-over feature quickly detects the failure and spawns a new nexus to ensure I/O continuity.

Reply:
Updated the text.